pkg/operation/botanist/etcd.go

```diff
@@ -51,9 +51,11 @@ func (b *Botanist) DefaultEtcd(role string, class etcd.Class) (etcd.Interface, e
 		return nil, err
 	}
 
-	var replicas *int32
+	var (
+		replicas *int32
+	)
```

Comment:
Typically, we only use blocks if there is more than one variable.
```suggestion
	var replicas *int32
```